test/loopback.test.js

```diff
@@ -631,4 +631,73 @@ describe('loopback', function() {
       });
     });
   });
+
+  describe('new remote method configuration', function() {
+    it('Remote method - isStatic specified in method name', function() {
+      var TestModel = loopback.createModel(uniqueModelName);
+      loopback.configureModel(TestModel, {
+        dataSource: null,
+        methods: {
+          staticMethod: {
+            http: { path: '/static' }
+          },
+          'prototype.instanceMethod': {
+            http: { path: '/instance' }
+          }
+        }
+      });
+
+      var methodNames = TestModel.sharedClass.methods().map(function(m) {
+        return m.stringName.replace(/^[^.]+\./, ''); // drop the class name
+      });
+
+      expect(methodNames).to.include.members([
+        'staticMethod',
+        'prototype.instanceMethod'
+      ]);
+    });
+
+    it('Incorrectly defined remote method', function() {
```

Comment:
Please make the test read as an English sentence, where the initial `it` stands for the name from the parent `describe` blocks, in this case `... new remote method configuration`. A good test name should describe 1) what scenario/conditions are tested 2) what is the expected outcome. That way the list of tests can be read as a functional specification.

For example:

```js
it('throws when "isStatic" is "true" and method name starts with "prototype."', ...);
```